### ISSUE TITLE
Feature request: act as account associated with column by default

### ISSUE THREAD

**Issue by ineffyble** [NicolasConstant/sengi#484]
This is one big behaviour difference between Sengi and Tweetdeck that trips me up.

In tweetdeck, if I add columns from multiple accounts, then my interactions in that column will be from the "owning" account. So I can like a tweet on the home feed of AccountA, as AccountA, and then like a tweet in the notifications of AccountB, as AccountB.

If Sengi could support this behaviour of defaulting the user context from the column, instead of the account switcher, that would be awesome.

**Comment by karlp** [comment]
Yeah, I've been running into this a lot too.  With two accounts (for two different languages) I have two columns, one for the local timeline on each server, and regularly have the "wrong" account selected on the far left :|
